frontend/zc_messaging/src/App.js

```diff
@@ -1,17 +1,11 @@
 import React from "react"
-import { Redirect, Route, Switch } from "react-router-dom"
-import AllDms from "./pages/all_dms/AllDms"
+import Routes from "./routes"
 
 // import css styles here
 import "bootstrap/dist/css/bootstrap.min.css"
 
 export default function App() {
   return (
-    <Switch>
-      <Route exact path="/all-dms" component={AllDms} />
-      <Route exact path="*">
-        <Redirect to="/all-dms" />
-      </Route>
-    </Switch>
+    <Routes />
```

Comment:
am not sure we should have this in a component, since App is our entry point then the route definitions should be here instead. or is there a reason for this

Reply:
The reason was just to separate routing logic form other business logic from App.js

It's basically the same thing

Reply:
i know it's the same thing but that means the app.js file might only contain that "route" component

Reply:
There is another route file for right sidebar components like thread, channel details, member profile

Reply:
okay